_includes/manuals/1.17/4.4.2_prov_media.md

```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. It is important to understand that, in the Red Hat World, installation media are a resource for the installation repo, not for the updates repo.
```

Comment:
How about: Make sure to use URL with OS installer images, some Linux distributions publish repositories containing no installers (e.g. fedora-updates).

Reply:
much better for me.

Reply:
>The Installation Media represents the web URL from where the installation packages
>can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when
>Foreman is first installed. However, it is best to edit the media you are going to
>use and ensure the Family is set. Make sure to use URLs that point to
>actual OS installer repositories. The installer-less repositories published by
>some Linux distributions (e.g. fedora-updates) won't work here

how about that?

Reply:
Better, thanks. Also "However, it is best to edit the media you are going to
use and ensure the Family is set." is a bit weird to me. Perhaps "Be sure to select closest or local mirror and ensure family is set correctly."